erts/doc/src/erl.xml

```diff
@@ -1633,8 +1633,7 @@
       <tag>The <c>.erlang</c> startup file</tag>
       <item>
         <p>When Erlang/OTP is started, the system searches for a file named
-          <c>.erlang</c> in the directory where Erlang/OTP is started. If not
-          found, the user's home directory is searched for an <c>.erlang</c>
+          <c>.erlang</c> in the user's home directory is searched for an <c>.erlang</c>
           file.</p>
```

Comment:
The latter half of this sentence can be dropped. It should read:

> When Erlang/OTP is started, the system searches for a file named `.erlang` in the user's home directory.

Reply:
Thanks, fixed.